key_files/totalbenca.yml

```diff
@@ -0,0 +1,17 @@
+---
+name            : TotalBenCa
+units           : Peq
+model           : MEDUSA
+modelgrid       : eORCA1
+dimensions      : 1
+modelFiles      : $BASEDIR_MODEL/$JOBID/medusa*_$JOBIDo_1y_*_diad-T.nc
+model_vars      : BEN_CA
+model_convert   :
+    path: bgcval2/functions/BenCa.py
```

Comment:
And this new file too! `bgcval2/functions/BenCa.py`.

If you run a `git status` command, it will show you all the files you've created but not included in the git repository!

Reply:
Added now